src/main/java/seedu/hirehub/logic/commands/SlotsLeftCommand.java

```diff
@@ -0,0 +1,67 @@
+package seedu.hirehub.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.hirehub.commons.core.index.Index;
+import seedu.hirehub.commons.util.ToStringBuilder;
+import seedu.hirehub.logic.Messages;
+import seedu.hirehub.logic.commands.exceptions.CommandException;
+import seedu.hirehub.model.Model;
+import seedu.hirehub.model.job.Job;
+
+/**
+ * Adds an job application from a candidate to the list of job applications
+ */
+public class SlotsLeftCommand extends Command {
+    public static final String COMMAND_WORD = "slots_left";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Retrieves the number of slots left for the selected"
+            + " job computed by subtracting the number of vacancy with the number of candidates who got accepted \n"
+            + "Parameters: INDEX (index of the job displayed in the job list which must be a positive integer)\n"
```

Comment:
To be clearer, the first part of the message should be "Retrieves the number of slots left for the selected job computed by subtracting the number of accepted candidates from total vacancies"